src/interchange/src/avro/envelope_debezium.rs

```diff
@@ -348,6 +340,29 @@ impl<'a> AvroDecode for DebeziumSourceDecoder<'a> {
                         DecodeError::Custom("\"lsn\" is not an integer".to_string())
                     })?);
                 }
+                "sequence" => {
+                    let next = ValueDecoder;
+                    let val = field.decode_field(next)?;
+                    let val = match val {
+                        Value::Union { inner, .. } => *inner,
+                        val => val,
+                    };
+                    let val = match val.into_string() {
+                        Some(val) => val,
+                        None => {
+                            return Err(AvroError::Decode(DecodeError::Custom(
+                                "\"sequence\" is not a string".to_string(),
+                            )))
+                        }
+                    };
+                    let seq: Vec<Option<String>> = serde_json::from_str(&val)
+                        .map_err(|e| AvroError::Decode(DecodeError::Custom(e.to_string())))?;
+                    sequence = Some(
+                        seq.into_iter()
+                            .map(|s| s.and_then(|s| Some(s.parse().unwrap())))
+                            .collect(),
```

Comment:
if somehow we got bad data from the sequence number would that crash materialized? Should we do something like:

```suggestion
                        seq.into_iter()
                            .map(|s| s.map(|s| s.parse()).transpose())
                            .collect::<Result<_, _>()
                            .map_err(AvroError::Custom(...))?,
```

Reply:
thank you!